analyses/sample-distribution-analysis/01-filter-across-types.R

```diff
@@ -0,0 +1,143 @@
+# This script filters the given dataset to produce a summarized visualization
+# of key variables within the dataset.
+#
+# Chante Bethell for CCDL 2019
+
```

Comment:
Can you add a statement about how this script is intended to be run? An example would be:

```
#### USAGE
This script is intended to be run via the command line from the top directory of the repository:
Rscript analyses/sample-distribution-analysis/01-filter-across-types.R
```